<p><b>[INFO]</b>&nbsp;GET Request : url=http://192.168.29.116/api/v1/hapi/DUT_A/chipcard_success/ <br> path_url=/api/v1/hapi/DUT_A/chipcard_success/ <br> headers={'User-Agent': 'python-requests/2.25.1', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Authorization': 'Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJmcmVzaCI6ZmFsc2UsImlhdCI6MTcyODkwNjA2MywianRpIjoiYWRmM2Y1ODctNGMzNi00YzQ1LTlmNDUtMTI1MTE3MzdiNzBkIiwidHlwZSI6ImFjY2VzcyIsInN1YiI6ImFkbWluQHNhc3RyYS5jb20iLCJuYmYiOjE3Mjg5MDYwNjMsImV4cCI6MTczMTQ5ODA2M30.qlpAIGSSDrHx4VSIQ2xjTTl7CH-NkKVhJIRLl2X01YM', 'Content-Type': 'application/x-www-form-urlencoded'} <br> body=None <br> </p><p><b>[INFO]</b>&nbsp;GET Response : url=http://192.168.29.116/api/v1/hapi/DUT_A/chipcard_success/ <br> status=500, reason=Internal Server Error <br> headers={'Server': 'nginx/1.18.0 (Ubuntu)', 'Date': 'Wed, 16 Oct 2024 08:27:17 GMT', 'Content-Type': 'text/html', 'Content-Length': '141', 'Connection': 'keep-alive'} <br> body=<html><br>  <head><br>    <title>Internal Server Error</title><br>  </head><br>  <body><br>    <h1><p>Internal Server Error</p></h1><br>    <br>  </body><br></html><br> <br> </p><p style="color: red"><b>[FAIL]</b>&nbsp;HTTPError: 500 Server Error: Internal Server Error for url: http://192.168.29.116/api/v1/hapi/DUT_A/chipcard_success/</p>